crates/hc/src/lib.rs

```diff
@@ -103,7 +103,6 @@ impl CliSubcommand {
             CliSubcommand::Dna(cmd) => cmd.run().await?,
             CliSubcommand::WebApp(cmd) => cmd.run().await?,
             CliSubcommand::Sandbox(cmd) => cmd.run().await?,
-            CliSubcommand::RunLocalServices(cmd) => cmd.run().await,
```

Comment:
Will local services be removed altogether or is this a temporary measure for the purpose of integration? I'm mainly asking from the perspective of the use of local services for the purpose of testing happs without need to use/clutter up production services.

Reply:
Removed entirely. The kitsune2 bootstrap server now also composes the sbd server, so you'll just need to run that one thing. We'll want to do a bunch of documentation as the last step before finally merging the integration branch into develop.